pkgs/applications/misc/octoprint/default.nix

```diff
@@ -74,13 +74,13 @@ let
           self: super: {
             octoprint = self.buildPythonPackage rec {
               pname = "OctoPrint";
-              version = "1.8.4";
+              version = "1.8.5";
 
               src = fetchFromGitHub {
                 owner = "OctoPrint";
                 repo = "OctoPrint";
                 rev = version;
-                hash = "sha256-oYP+K7WBkYP7gajXZdbZso17u+GeyrIgEbhNjwRXbAo=";
+                hash = "sha256-dwDx8ktUeva7a1AZJuIdsk4GvZGm3HNDXfsVR0OuscU=";
```

Comment:
When trying to build I get the following error:

```
error: hash mismatch in fixed-output derivation '/nix/store/hk89fyq6jq736rx1p8f345lk3b09x2ph-source.drv':
         specified: sha256-dwDx8ktUeva7a1AZJuIdsk4GvZGm3HNDXfsVR0OuscU=
            got:    sha256-6KWkoLTuhYdwH3oi3mFtXascHqQ1cdL6oPx28/dczyM=
error: 1 dependencies of derivation '/nix/store/kz9zna2wg8fqa38cxhmc07zi0xcmsx1x-python3.10-OctoPrint-1.8.5.drv' failed to build
error: 1 dependencies of derivation '/nix/store/spfgkrvwrl33vhh6dwlizdlrp4hzmv9x-review-shell.drv' failed to build
```

Reply:
I can look into this later. Maybe they force-pushed on the tag..

Reply:
I works now, maybe it was a problem with my connection.